protoc

```diff
@@ -14,10 +14,13 @@ protocversion=3.0.0
 protocurl="https://github.com/google/protobuf/releases/download/v${protocversion}/protoc-${protocversion}-${os}-${arch}.zip"
 
 if [ ! -f "$protocbin" ]; then
-  echo "downloading $protocbin" >&2
+  echo "downloading $protocbin"
```

Comment:
Ah, yeah, that makes sense. I'm inclined to just remove this `echo` from the script. Does that work for folks?